tools/attrd_updater.c

```diff
@@ -255,6 +256,8 @@ main(int argc, char **argv)
         exit_code = CRM_EX_USAGE;
         g_set_error(&error, PCMK__EXITC_ERROR, exit_code, "Command requires --name argument");
         goto done;
+    } else if ((options.command == 'B'|| options.command == 'Y') && options.attr_dampen == NULL) {
+        fprintf(stderr, "Warning: '%c' command given without required --delay\n", options.command);
```

Comment:
Why does attrd_updater still use fprintf()?

Reply:
It certainly shouldn't.  I'll fix this usage and then there's two "Could not connect to attrd" strings that I'll fix in a new patch.

Reply:
Updated and pushed.